<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyDAQ</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon.svg</normaloff>:/icon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,50">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="configurationGroupBox">
          <property name="title">
           <string>Configuration</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="configureButton">
             <property name="toolTip">
              <string>Configure [Ctrl+C]</string>
             </property>
             <property name="text">
              <string>Configure</string>
             </property>
             <property name="shortcut">
              <string>Ctrl+C</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="newFileButton">
             <property name="toolTip">
              <string>New file [Ctrl+N]</string>
             </property>
             <property name="toolTipDuration">
              <number>-1</number>
             </property>
             <property name="text">
              <string>New file</string>
             </property>
             <property name="shortcut">
              <string>Ctrl+N</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="displayGroupBox">
          <property name="title">
           <string>Display</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="playButton">
             <property name="toolTip">
              <string>Start [P]</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/play.svg</normaloff>
               <normalon>:/pause.svg</normalon>:/play.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="shortcut">
              <string>P</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="stopButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Stop [S]</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/stop.svg</normaloff>:/stop.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="shortcut">
              <string>S</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="captureGroupBox">
          <property name="title">
           <string>Capture</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="recordButton">
               <property name="toolTip">
                <string>Capture data to file [C]</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/rec.svg</normaloff>:/rec.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="shortcut">
                <string>C</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="stopRecordButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Stop [S]</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/stop.svg</normaloff>:/stop.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="shortcut">
                <string>S</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="videoCheckBox">
             <property name="text">
              <string>Video</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="markersGroupBox">
          <property name="title">
           <string>Markers</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QPushButton" name="addMarkerButton">
             <property name="toolTip">
              <string>Add marker [M]</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/add.svg</normaloff>:/add.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="shortcut">
              <string>M</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="editMarkerButton">
             <property name="toolTip">
              <string>Edit marker(s) [E]</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/edit.svg</normaloff>:/edit.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="shortcut">
              <string>E</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="quitButton">
            <property name="toolTip">
             <string>Quit [Ctrl+Q]</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/quit.svg</normaloff>:/quit.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Ctrl+Q</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="GraphicsLayoutWidget" name="graphicsView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
